_content/articles/notes-on-the-udana_ireland.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "Notes on the Udana"
+authors:
+  - ireland
+external_url: "https://journal.equinoxpub.com/BSR/article/view/15266/23244"
+drive_links:
+  - "https://drive.google.com/file/d/1RGUtfoRvzvG5wh9EI_sNc0REd9P315wu/view?usp=sharing"
```

Comment:
This needs to be the new link: https://drive.google.com/file/d/1CaGZC8D6v-O8ra2yLb-4WbEQscEG1Arj/view?usp=drivesdk